lib/file-view.coffee

```diff
@@ -19,7 +19,11 @@ class FileView extends HTMLElement
     @fileName.dataset.name = @file.name
     @fileName.dataset.path = @file.path
 
-    @fileName.classList.add(FileIcons.getService().iconClassForPath(@file.path))
+    iconClass = FileIcons.getService().iconClassForPath(@file.path, this)
```

Comment:
@as-cii Do you think a cleaner approach would be to pass an object like this as the second argument...?

```
{
	context: "tab",
	target: this
}
```